jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/core/JDTUtils.java

```diff
@@ -634,17 +635,22 @@ public static boolean hasLeadingSlash(String uriString) {
     /**
      * Returns a list of all accessors (getter and setter) of the given field.
      * 
-     * @param field
+     * @param unit, field
      * @return a list of accessor methods
      * @throws JavaModelException
      */
-    public static List<IMethod> getAccessors(IField field) throws JavaModelException {
-        List<IMethod> accessors = new ArrayList<>();
-        IMethod getter = GetterSetterUtil.getGetter(field);
-        IMethod setter = GetterSetterUtil.getSetter(field);
-        for (IMethod accessor : new IMethod[] { getter, setter }) {
-            if (accessor != null) {
-                accessors.add(accessor);
+    public static List<IMethod> getFieldAccessors(ICompilationUnit unit, IField field) throws JavaModelException {
+        List<IMethod> accessors = new ArrayList<IMethod>();
+        String fieldName = field.getElementName();
+        fieldName = fieldName.substring(0, 1).toUpperCase() + fieldName.substring(1);
+        String getterName = "get" + fieldName;
+        String setterName = "set" + fieldName;
```

Comment:
We should consider the case in which the getter name is "isField" for a boolean field. There was a relevant discussion [here](https://github.com/eclipse/lsp4jakarta/pull/270#discussion_r829438115). We could possibly continue using Util methods.

Reply:
Done